lua/nui/popup/popup.md

```diff
@@ -0,0 +1,460 @@
+# Popup
```

Comment:
Rename the file to `README.md`. Otherwise GitHub won't render the markdown file when you're in `lua/nui/popup` folder:

![image](https://user-images.githubusercontent.com/8050659/130230407-452db9d0-3b7f-47d5-ba6a-05e1939c879b.png)

Renaming it to `README.md` will render the markdown file below the file list... just like you see it on the homepage:

![image](https://user-images.githubusercontent.com/8050659/130230543-48810d33-0ada-47cc-803d-fc625c26c450.png)